### DISCUSSION TITLE
How to Make and publish a CLI tool to jsr

### DISCUSSION THREAD

**Discussion by MikeyA-yo** [denoland/deno#23408] [Q&A] [answered]
i'm making a CLI tool, that i want to publish into JSR, any ideas on how to do that??

**Comment by bartlomieju** [comment]
You publish it as a regular package, specify a config like so:
```
{
  "name": "@scope/name",
  "version": "0.0.1",
  "exports": "./path/to/entrypoint.ts"
}
```

Then you can run it as: `deno run jsr:@scope/name`

**Reply by MikeyA-yo** [reply]
ok so how does an end user use it like install it globally on their system (if that's possible, if not fine)

**Reply by bartlomieju** [reply, marked as answer]
This should do the trick:
```
deno install --global jsr:@scope/name
```
